src/Generators/RoomGenerator.cs

```diff
@@ -47,7 +47,10 @@ public Room GenerateRoom()
 			var pivot = lastRoom.ExitPoint + new Point (1, -offset);
 
 			var room = GenerateEmpty (width, height, pivot, lastRoom.ExitPoint, !first);
-
+            var gen = new CharacterGenerator();
+            var character = gen.GenerateCharacter();
+            character.Transform.Position = room.Position + new Point(room.Size.X / 2, room.Size.Y /2);
```

Comment:
tak wiem to zostało dodane tylko aby sprawdzić czy poprawnie działa i jako przykład uzycia